web/components/og/og-market.tsx

```diff
@@ -0,0 +1,177 @@
+import { OgCardProps } from 'common/contract-details'
+import clsx from 'clsx'
+import { Sparkline, ogPoint } from './graph'
+import { formatPercent } from 'common/util/format'
+
+// Notes for working with this:
+// - Some css elements are missing or broken (e.g. 'gap' and 'text-ellipsis' and 'line-clamp')
+// - I also can't make things overflow hidden in only one direction
+// - Every element should have `flex` set
```

Comment:
FYI, I know this comment says this stuff, but I don't really trust it. I would trust the actual documentation: https://github.com/vercel/satori

So for example I bet that `gap` works (which would let you simplify some things) and I bet that `text-ellipsis` works.

Maybe I am being too cynical, I just think that if this comment author had known what they were talking about, they would have at least linked to the docs.

Reply:
gap didn't work when I tried. I'll look into it